test/unit/geo/transform.test.js

```diff
@@ -729,13 +728,36 @@ test('transform', (t) => {
         const zoom = transform._zoomFromMercatorZ(altitudeZ);
         transform.zoom = zoom;
 
-        // Pitch should have been adjusted so that the camera isn't under the terrain
-        const pixelsPerMeter = mercatorZfromAltitude(1, transform.center.lat) * transform.worldSize;
-        const updatedAltitude = transform.cameraToCenterDistance / pixelsPerMeter * Math.cos(degToRad(transform.pitch));
+        const cameraAltitude = altitudeFromMercatorZ(transform.getFreeCameraOptions().position.z, transform.getFreeCameraOptions().position.y);
+        transform.updateElevation(false);
```

Comment:
Missing second argument here?
```suggestion
        transform.updateElevation(false, false);
```

Reply:
Strange I have this locally! Thanks for the catch

Reply:
I'll update with a default value for isDragging so this won't be necessary! Thanks for the catch!